### PR TITLE
Papa 11 deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates to HYPM, FLMA/B for deployments and recoveries.  Per Dee, "I highlighted the only instruments that were swapped between FLMA & B.  ADCP’s and PHSEN’s were reversed from the original plan during the SMC burn in.  No other instrument changes occurred."

Because of the swap of the ADCPs on FLMA & FLMB, I also had to update the associated cruise-specific ADCP cal csvs (just updating the mooring assignments).

[GP02HYPM-00011-6-2-2024_DE.xlsx](https://github.com/user-attachments/files/15829095/GP02HYPM-00011-6-2-2024_DE.xlsx)
[GP03FLMA-00011-2024-06-13_DE.xlsx](https://github.com/user-attachments/files/15829096/GP03FLMA-00011-2024-06-13_DE.xlsx)
[GP03FLMB-00011_2024-06-13_DE.xlsx](https://github.com/user-attachments/files/15829097/GP03FLMB-00011_2024-06-13_DE.xlsx)
[QLR-Mooring Tables.docx](https://github.com/user-attachments/files/15829098/QLR-Mooring.Tables.docx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Papa 11 AM Updates](https://trello.com/c/oKNl8zE4/238-papa-11-am-updates)

File: deployment/GP02HYPM_Deploy.csv [hunk 1]
```diff
@@ -119,31 +119,31 @@ SKQ202208S,,SKQ202308S,,GP02HYPM-WFP03-01-FLORDL000,9,1,2022-05-17T20:09:00,2023
...
+SKQ202408S,,,,GP02HYPM-WFP02-04-CTDPFL000,11,1,2024-06-02T20:23:00,,CGMGP-02HYPM-00011,CGVEH-GWFPGX-99701,CGINS-CTDPFL-00115,50.1143,-144.91743,,2090,4237,,,E
+SKQ202408S,,,,GP02HYPM-WFP02-05-VEL3DL000,11,1,2024-06-02T20:23:00,,CGMGP-02HYPM-00011,CGVEH-GWFPGX-99701,CGINS-VEL3DL-01026,50.1143,-144.91743,,2090,4237,,,E
+SKQ202408S,,,,GP02HYPM-WFP03-00-WFPENG000,11,1,2024-06-02T20:23:00,,CGMGP-02HYPM-00011,CGVEH-GWFPGX-10403,CGCON-GWFPGX-10403,50.1143,-144.91743,,4063,4237,,,E
+SKQ202408S,,,,GP02HYPM-WFP03-01-FLORDL000,11,1,2024-06-02T20:23:00,,CGMGP-02HYPM-00011,CGVEH-GWFPGX-10403,CGINS-FLORDL-03284,50.1143,-144.91743,,4063,4237,,,E
+SKQ202408S,,,,GP02HYPM-WFP03-03-DOSTAL000,11,1,2024-06-02T20:23:00,,CGMGP-02HYPM-00011,CGVEH-GWFPGX-10403,CGINS-DOSTAL-01478,50.1143,-144.91743,,4063,4237,,,E
+SKQ202408S,,,,GP02HYPM-WFP03-04-CTDPFL000,11,1,2024-06-02T20:23:00,,CGMGP-02HYPM-00011,CGVEH-GWFPGX-10403,CGINS-CTDPFL-00132,50.1143,-144.91743,,4063,4237,,,E
+SKQ202408S,,,,GP02HYPM-WFP03-05-VEL3DL000,11,1,2024-06-02T20:23:00,,CGMGP-02HYPM-00011,CGVEH-GWFPGX-10403,CGINS-VEL3DL-01115,50.1143,-144.91743,,4063,4237,,,E
```
Comment (cooleyky) [inline deployment/GP02HYPM_Deploy.csv:149]: looks good

File: deployment/GP03FLMA_Deploy.csv [hunk 2]
```diff
@@ -161,39 +161,39 @@ SKQ202208S,,SKQ202308S,,GP03FLMA-RIS01-00-SIOENG000,9,1,2022-05-19T17:27:00,2023
...
+SKQ202408S,,,,GP03FLMA-RIM01-02-CTDMOG048,11,1,2024-06-05T20:03:00,,CGMGP-03FLMA-00011,,CGINS-CTDMOG-11647,50.02322,-144.36115,,500,4104,,,C
+SKQ202408S,,,,GP03FLMA-RIM01-02-CTDMOH049,11,1,2024-06-05T20:03:00,,CGMGP-03FLMA-00011,,CGINS-CTDMOH-10228,50.02322,-144.36115,,750,4104,,,C
+SKQ202408S,,,,GP03FLMA-RIM01-02-CTDMOH050,11,1,2024-06-05T20:03:00,,CGMGP-03FLMA-00011,,CGINS-CTDMOH-11601,50.02322,-144.36115,,1000,4104,,,C
+SKQ202408S,,,,GP03FLMA-RIM01-02-CTDMOH051,11,1,2024-06-05T20:03:00,,CGMGP-03FLMA-00011,,CGINS-CTDMOH-11602,50.02322,-144.36115,,1500,4104,,,C
+SKQ202408S,,,,GP03FLMA-RIS01-00-SIOENG000,11,1,2024-06-05T20:03:00,,CGMGP-03FLMA-00011,,CGCON-SIOCON-13995,50.02322,-144.36115,,30,4104,,,C
+SKQ202408S,,,,GP03FLMA-RIS01-03-DOSTAD000,11,1,2024-06-05T20:03:00,,CGMGP-03FLMA-00011,,CGINS-DOSTAD-00460,50.02322,-144.36115,,30,4104,,,C
+SKQ202408S,,,,GP03FLMA-RIS01-04-PHSENF000,11,1,2024-06-05T20:03:00,,CGMGP-03FLMA-00011,,CGINS-PHSENF-P0147,50.02322,-144.36115,,30,4104,,,C
+SKQ202408S,,,,GP03FLMA-RIS01-05-FLORTD000,11,1,2024-06-05T20:03:00,,CGMGP-03FLMA-00011,,CGINS-FLORTD-01300,50.02322,-144.36115,,30,4104,,,C
```
Comment (cooleyky) [inline deployment/GP03FLMA_Deploy.csv:199]: looks good

File: deployment/GP03FLMB_Deploy.csv [hunk 3]
```diff
@@ -161,39 +161,39 @@ SKQ202208S,,SKQ202308S,,GP03FLMB-RIS01-00-SIOENG000,9,1,2022-05-18T19:57:00,2023
...
+SKQ202408S,,,,GP03FLMB-RIM01-02-CTDMOG068,11,1,2024-06-07T04:00:00,,CGMGP-03FLMB-00011,,CGINS-CTDMOG-11610,50.37608,-144.51397,,500,4175,,,C
+SKQ202408S,,,,GP03FLMB-RIM01-02-CTDMOH069,11,1,2024-06-07T04:00:00,,CGMGP-03FLMB-00011,,CGINS-CTDMOH-11680,50.37608,-144.51397,,750,4175,,,C
+SKQ202408S,,,,GP03FLMB-RIM01-02-CTDMOH070,11,1,2024-06-07T04:00:00,,CGMGP-03FLMB-00011,,CGINS-CTDMOH-11681,50.37608,-144.51397,,1000,4175,,,C
+SKQ202408S,,,,GP03FLMB-RIM01-02-CTDMOH071,11,1,2024-06-07T04:00:00,,CGMGP-03FLMB-00011,,CGINS-CTDMOH-11682,50.37608,-144.51397,,1500,4175,,,C
+SKQ202408S,,,,GP03FLMB-RIS01-00-SIOENG000,11,1,2024-06-07T04:00:00,,CGMGP-03FLMB-00011,,CGCON-SIOCON-14884,50.37608,-144.51397,,30,4175,,,C
+SKQ202408S,,,,GP03FLMB-RIS01-03-DOSTAD000,11,1,2024-06-07T04:00:00,,CGMGP-03FLMB-00011,,CGINS-DOSTAD-00462,50.37608,-144.51397,,30,4175,,,C
+SKQ202408S,,,,GP03FLMB-RIS01-04-PHSENF000,11,1,2024-06-07T04:00:00,,CGMGP-03FLMB-00011,,CGINS-PHSENF-P0104,50.37608,-144.51397,,30,4175,,,C
+SKQ202408S,,,,GP03FLMB-RIS01-05-FLORTD000,11,1,2024-06-07T04:00:00,,CGMGP-03FLMB-00011,,CGINS-FLORTD-01301,50.37608,-144.51397,,30,4175,,,C
```
Comment (cooleyky) [inline deployment/GP03FLMB_Deploy.csv:199]: looks good

**Review by cooleyky — APPROVED** [review]
deployment and recovery info looks good!

File: calibration/ADCPSL/CGINS-ADCPSL-18261__20240528.csv
Comment (reedan88) [inline calibration/ADCPSL/CGINS-ADCPSL-18261__20240528.csv:1]: Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-20502__20240528.csv
Comment (reedan88) [inline calibration/ADCPSL/CGINS-ADCPSL-20502__20240528.csv:1]: Looks good.

File: deployment/GP02HYPM_Deploy.csv
Comment (reedan88) [inline deployment/GP02HYPM_Deploy.csv:1]: Looks good.

File: deployment/GP03FLMA_Deploy.csv
Comment (reedan88) [inline deployment/GP03FLMA_Deploy.csv:1]: Looks good.

File: deployment/GP03FLMB_Deploy.csv
Comment (reedan88) [inline deployment/GP03FLMB_Deploy.csv:1]: Looks good.

File: deployment/GP05MOAS-GL276_Deploy.csv
Comment (reedan88) [inline deployment/GP05MOAS-GL276_Deploy.csv:1]: Looks good.

File: deployment/GP05MOAS-GL363_Deploy.csv
Comment (reedan88) [inline deployment/GP05MOAS-GL363_Deploy.csv:1]: Looks good.

**Review by reedan88 — APPROVED** [review]

**Pull request merged**